core/internal/gql/gql_gen.go

```diff
@@ -747,6 +747,21 @@ func (v *ServerInfoServerInfoLatestLocalVersionInfo) GetVersionOnThisInstanceStr
 	return v.VersionOnThisInstanceString
 }
 
+type TagInput struct {
+	TagCategoryName *string `json:"tagCategoryName"`
```

Comment:
Can you explain how the `tagCategoryName` is different from `tagName`?

Reply:
It's not currently used in practice -- but `tagCategoryName` was included in the `Tag` schema back when tags were originally implemented for artifact collections.

IIUC, the original idea was to support grouping different "types" of tags -- e.g. if `tagCategoryName == "dataset"`, there could conceivably be `dataset:train` (i.e. `tagName == "train"`), `dataset:test` (i.e. `tagName == "test"`), etc.

(incidentally, this was/is the reason `:` characters are disallowed in tag names)